src/roles/vsd-health/tasks/main.yml

```diff
@@ -406,44 +398,152 @@
+    register: keystore_pass_check
+    when: keyStorePwd is defined
+
+  - name: Write keystore password check separator to report file
+    nuage_append: filename="{{ report_path }}" text="-----Results of verifying if keystore password is set-----\n"
+    delegate_to: localhost
+    when: keyStorePwd is defined
+
+  - name: Write keystore password check result
+    nuage_append: filename="{{ report_path }}" text="{{ keystore_pass_check | dict2items | rejectattr("key", "eq", "cmd") | list | items2dict | to_nice_json }}\n"
+    delegate_to: localhost
     when: keyStorePwd is defined
 
   - name: Verify truststore pass is set
     command: "keytool -keystore /opt/vsd/jboss/standalone/configuration/vsd.truststore -storepass {{ trustStorePwd }} -list"
+    register: truststore_pass_check
+    when: trustStorePwd is defined
+
+  - name: Write truststore password check separator to report file
+    nuage_append: filename="{{ report_path }}" text="-----Results of verifying truststore password-----\n"
+    delegate_to: localhost
+    when: trustStorePwd is defined
+
+  - name: Write truststore password check result
+    nuage_append: filename="{{ report_path }}" text="{{ truststore_pass_check | dict2items | rejectattr("key", "eq", "cmd") | list | items2dict | to_nice_json }}\n"
+    delegate_to: localhost
     when: trustStorePwd is defined
 
   - name: Verify keyserver pass is set
     command: "keytool -keystore /opt/vsd/jboss/standalone/configuration/keyserver.jks -storepass {{ keyServerStorePwd }} -list"
+    register: keyserver_pass_check
+    when: keyServerStorePwd is defined
+  
+  - name: Write keyserver password check separator to report file
+    nuage_append: filename="{{ report_path }}" text="-----Results of verifying keyserver password-----\n"
+    delegate_to: localhost
+    when: keyServerStorePwd is defined
+
+  - name: Write keyserver password check result
+    nuage_append: filename="{{ report_path }}" text="{{ keyserver_pass_check | dict2items | rejectattr("key", "eq", "cmd") | list | items2dict | to_nice_json }}\n"
+    delegate_to: localhost
     when: keyServerStorePwd is defined
 
   - name: Verify jrestore pass is set for version
     command: "keytool -keystore {{ java_home.stdout }}/lib/security/cacerts -alias vspca -storepass {{ jreStorePwd }} -list"
+    register: jrestore_pass_check
+    when:
+      - jreStorePwd is defined
+      - vsd_major_version|int < 6
+
+  - name: Write jrestore password check separator to report file
+    nuage_append: filename="{{ report_path }}" text="-----Results of verifying jrestore password-----\n"
+    delegate_to: localhost
+    when:
+      - jreStorePwd is defined
+      - vsd_major_version|int < 6
+
+  - name: Write jrestore password check result
+    nuage_append: filename="{{ report_path }}" text="{{ jrestore_pass_check | dict2items | rejectattr("key", "eq", "cmd") | list | items2dict | to_nice_json }}\n"
```

Comment:
this will filter out the "cmd" attribute so that the password does not get written to the report